src/components/FileRepo/AggregationSidebar/AggregationSidebar.js

```diff
@@ -91,34 +94,44 @@ const AggregationSidebar = compose(injectState, withTheme)(
                         ...props,
                         translateSQONValue,
                         closeModal: effects.unsetModal,
+                        onFacetNavigation: path => {
+                          let facetNavEvent = {
+                            category: TRACKING_EVENTS.categories.fileRepo.filters + ' - Advanced',
+                            action: TRACKING_EVENTS.actions.click + ' side navigation',
+                            label: path,
+                          };
+                          trackFileRepoInteraction(facetNavEvent);
```

Comment:
doesn't seem it's breaking anything here but for safety, it probably should be standard to check on all these callbacks coming from props to call them as well when it's spread down like here, otherwise this would override anything passed down from above